dev_guide/pod_autoscaling.adoc

```diff
@@ -92,8 +92,8 @@ are given default values from the {product-title} server.
 For example:
 
 ----
-$ oc autoscale dc/frontend --min 1 --max 10 /
-  --cpu-percent=80 deploymentconfig "frontend" autoscaled
+$ oc autoscale dc/frontend --min 1 --max 10 / --cpu-percent=80 
+deploymentconfig "frontend" autoscaled
```

Comment:
@tmorriso-rh @ghyde Do we need the `/` between `--max 10` `--cpu-percent=80`?

Reply:
No. My PR (#6081) doesn't have that.